openedx/core/djangoapps/content_libraries/api.py

```diff
@@ -167,6 +167,7 @@ class ContentLibraryMetadata:
     # Allow any user with Studio access to view this library's content in
     # Studio, use it in their courses, and copy content out of this library.
     allow_public_read = attr.ib(False)
+    can_edit_library = attr.ib(False)
```

Comment:
This is a very significant change, because until this field was added, the metadata here was all user-agnostic. Making it specific to a user completely changes how it can be used. I'm going to suggest that we don't add this field here (but we do add it to the serializer).

Reply:
I understand. Updated: https://github.com/openedx/edx-platform/pull/34934/commits/c303069324b444647ac3006fee882ba42d7f42e7#diff-a760c3e87f2f671ac2453abbbaf6f10ca052c7acf1c46c1d36f24646e1d52a00L170